src/config.ts

```diff
@@ -8,8 +8,12 @@ const EPSAGON_APP_NAME      = process.env.EPSAGON_APP_NAME || 'CarsImport'
 
 const DB_ENDPOINT = encodeURI(`postgresql://${DB_USERNAME}:${DB_PASSWORD}@${DB_ADDRESS}:${DB_PORT}/${DB_NAME}`)
 
+const EPSAGON = {
+  APP_NAME: 'CarsImport',
```

Comment:
All epsagon app names in integration are the same, we do not distinguish between stacks. `volta-integration` and `volta-production` are all we have at the moment. 
```suggestion
  APP_NAME: `volta-${ENV}`,
```

Reply:
Sorry, this is not current anymore. This APP_NAME is redundant and can be replaced in the epsagon.init with the above \`volta-${ENV}\`. The only var we need to import is the epsagon token.

Reply:
I'll also make this change across the other stacks